### PR TITLE
Fixed #32202 -- Fixed autoreloader argument generation on Windows with Python 3.7-.

### PR DESCRIPTION
**Pull request by carltongibson**

### CODE REVIEW

File: django/utils/autoreload.py [hunk 1]
```diff
@@ -232,11 +232,15 @@ def get_child_arguments():
...
-            return [exe_entrypoint, *sys.argv[1:]]
+            # TODO: Remove str() when dropping support for PY37.
+            # arg parameter accepts path-like on Windows from Python 3.8.
+            return [str(exe_entrypoint), *sys.argv[1:]]
         script_entrypoint = py_script.with_name('%s-script.py' % py_script.name)
         if script_entrypoint.exists():
             # Should be executed as usual.
-            return [*args, script_entrypoint, *sys.argv[1:]]
+            # TODO: Remove str() when dropping support for PY37.
+            # arg parameter accepts path-like on Windows from Python 3.8.
```
Comment (felixxm) [inline django/utils/autoreload.py:242]:
```suggestion
            # args parameter accepts path-like on Windows from Python 3.8.
```
The first `subprocess.run()`'s parameter is called [`args`](https://docs.python.org/3/library/subprocess.html#subprocess.run).

File: docs/releases/3.1.4.txt [hunk 2]
```diff
@@ -17,3 +17,6 @@ Bugfixes
 
 * Fixed crash of key transforms for :class:`~django.db.models.JSONField` on
   PostgreSQL when using on a ``Subquery()`` annotation (:ticket:`32182`).
+
+* Fixed a crash in Django 3.1 for certain invocations of ``runserver`` on
```
Comment (felixxm) [inline docs/releases/3.1.4.txt:21]: Maybe:
```
Fixed a regression in Django 3.1 that caused a crash of auto-reloader for ...
```

**Review by felixxm — APPROVED** [review]
@carltongibson Thanks :+1:

**Comment by carltongibson** [comment]
Thanks @felixxm!

**Pull request merged**